api/v1/server.gotmpl

```diff
@@ -646,22 +584,3 @@ func (s *Server) TLSListener() (net.Listener, error) {
 	return s.httpsServerL, nil
 }
 
-func handleInterrupt(once *sync.Once, s *Server) {
```

Comment:
I've removed the signal handling as it's now done by hive in both the agent and the operator, and I checked that hubble-relay which uses the health server also does signal handling itself.